package.json

```diff
@@ -106,6 +123,11 @@
                 "key": "f8",
                 "mac": "f8",
                 "when": "editorTextFocus"
+            },
+            {
+                "command": "ecl.showAllDocumentation",
+                "key": "f12",
+                "mac": "f12"
```

Comment:
Yeah, I felt F12 was the best key for this by default. If it gets in the way maybe we just remove the shortcut here but keep the command pallet option where you manually select it from CTRL+P and hit enter.